### PR TITLE
Add documentation for get embedding operation in Azure OpenAI

### PR DESCRIPTION
**Pull request by shivam-51**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Add documentation for get embedding operation in Azure OpenAI

## Issue reference

https://github.com/dapr/components-contrib/issues/3018

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/openai.md [hunk 1]
```diff
@@ -228,6 +229,53 @@ The response body contains the following JSON:
...
+  {{% /codetab %}}
+
+{{< /tabs >}}
+
+#### Response
+
+The response body contains the following JSON:
+
+```json
+[0.018574921,-0.00023652936,-0.0057790717,.... (1536 floats total for ada)]
```
Comment (shivam-51) [inline daprdocs/content/en/reference/components-reference/supported-bindings/openai.md:276]: Can someone help me fix the formatting here?

File: daprdocs/content/en/reference/components-reference/supported-bindings/openai.md [hunk 2]
```diff
@@ -228,6 +229,53 @@ The response body contains the following JSON:
 
 ```
 
+### Get Embedding API
+
+The `get-embedding` operations returns a vector representation of a given input that can be easily consumed by machine learning models and other algorithms.
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-bindings/openai.md]:
```suggestion
The `get-embedding` operation returns a vector representation of a given input that can be easily consumed by machine learning models and other algorithms.
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/openai.md [hunk 3]
```diff
@@ -228,6 +229,53 @@ The response body contains the following JSON:
...
+  {{% codetab %}}
+  ```bash
+curl -d '{
+    "data": {
+        "deploymentId": "embeddings",
+        "message": "The capital of France is Paris."
+    },
+    "operation": "get-embedding"
+}' \
+http://localhost:<darpr-port>/v1.0/bindings/<binding-name>
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-bindings/openai.md]:
```suggestion
http://localhost:<dapr-port>/v1.0/bindings/<binding-name>
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/openai.md [hunk 4]
```diff
@@ -228,6 +229,53 @@ The response body contains the following JSON:
...
+The data parameters are:
+
+- `deploymentId` - string that specifies the model deployment ID to use.
+- `message` - string that specifies the text to embed.
+
+#### Example
+
+{{< tabs Linux >}}
+
+  {{% codetab %}}
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-bindings/openai.md]: I don't think it makes a difference, but these don't need to be indented

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
initial review

File: daprdocs/content/en/reference/components-reference/supported-bindings/openai.md (hunk 2 above)
Comment (shivam-51) [reply daprdocs/content/en/reference/components-reference/supported-bindings/openai.md]: Done.

File: daprdocs/content/en/reference/components-reference/supported-bindings/openai.md (hunk 4 above)
Comment (shivam-51) [reply daprdocs/content/en/reference/components-reference/supported-bindings/openai.md]: Removed indentation.

File: daprdocs/content/en/reference/components-reference/supported-bindings/openai.md (hunk 3 above)
Comment (shivam-51) [reply daprdocs/content/en/reference/components-reference/supported-bindings/openai.md]: Fixed.

**Comment by shivam-51** [comment]
@hhunter-ms Please have a look.

**Review by hhunter-ms — APPROVED** [review]

**Comment by msfussell** [comment]
@shivam-51 - You also need to update the binding component file, so that this is listed as an alpha component in the Azure table here https://docs.dapr.io/reference/components-reference/supported-bindings/. Can you please add this?

**Comment by shivam-51** [comment]
> @shivam-51 - You also need to update the binding component file, so that this is listed as an alpha component in the Azure table here https://docs.dapr.io/reference/components-reference/supported-bindings/. Can you please add this?

@msfussell  That's already done in a previous PR https://v1-12.docs.dapr.io/reference/components-reference/supported-bindings/

**Comment by msfussell** [comment]
@shivam-51 - Great did not see that. BTW see this isssue https://github.com/dapr/community/issues/331 for the community call this Wed which I am hosting. Reach out to me on Discord or email.

**Comment by msfussell** [comment]
@shivam-51 - Please reach out to me for the Dapr community call tomorrow. I need to confirm you can present.

**Pull request merged**